main.py

```diff
@@ -1,10 +1,53 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as f:
+        date = json.loads(f.read())
+
+    def create_race(date: dict) -> Race:
+        date = date.get("race")
```

Comment:
I don't really like the usage of `.get()` here, since it's possible to use `[]`, at least in most cases.
It applies to the entire file, not just this line.

Reply:
It's my own prefer to use .get(), because you can indicate value when dict don't have some key, so I always use it.